src/store/modules/web3.ts

```diff
@@ -260,6 +265,22 @@ const actions = {
       return Promise.reject();
     }
   },
+  getBalances: async ({ commit, state }) => {
+    const spacesList = Object.values(spaces);
+
+    const balances = await Promise.all(
+      spacesList.map(space => getBalance(web3, space.token, state.account))
```

Comment:
We fetch the balance of `space.token`. Is this what we want or should we check against a different list?

Reply:
We fetch against `space.token` but we want to use a `multicall` like this https://github.com/balancer-labs/snapshot/blob/d91369d75d4659f05dd683dfd6eb10097422a129/src/store/modules/web3.ts#L311-L317 this way we not sending 1 request for each space.